e2e/interchain_accounts_test.go

```diff
@@ -389,3 +389,165 @@ func (s *InterchainAccountsTestSuite) TestMsgSubmitTx_SuccessfulBankSend_Incenti
 		})
 	})
 }
+
+func (s *InterchainAccountsTestSuite) TestMsgSubmitTx_FailedBankSend_Incentivized() {
+	t := s.T()
+	ctx := context.TODO()
+
+	// setup relayers and connection-0 between two chains
+	// channel-0 is a transfer channel but it will not be used in this test case
+	relayer, _ := s.SetupChainsRelayerAndChannel(ctx)
+	chainA, chainB := s.GetChains()
+
+	var (
+		chainADenom   = chainA.Config().Denom
+		interchainAcc = ""
+		testFee       = testvalues.DefaultFee(chainADenom)
+	)
+
+	t.Run("relayer wallets recovered", func(t *testing.T) {
+		err := s.RecoverRelayerWallets(ctx, relayer)
+		s.Require().NoError(err)
+	})
+
+	chainARelayerWallet, chainBRelayerWallet, err := s.GetRelayerWallets(relayer)
+	t.Run("relayer wallets fetched", func(t *testing.T) {
+		s.Require().NoError(err)
+	})
+
+	s.Require().NoError(test.WaitForBlocks(ctx, 5, chainA, chainB), "failed to wait for blocks")
+
+	chainARelayerUser, chainBRelayerUser := s.GetRelayerUsers(ctx)
+	relayerAStartingBalance, err := s.GetChainANativeBalance(ctx, chainARelayerUser)
+	s.Require().NoError(err)
+	t.Logf("relayer A user starting with balance: %d", relayerAStartingBalance)
+
+	controllerAccount := s.CreateUserOnChainA(ctx, testvalues.StartingTokenAmount)
+	chainBAccount := s.CreateUserOnChainB(ctx, testvalues.StartingTokenAmount)
+
+	t.Run("register interchain account", func(t *testing.T) {
+		version := "" // allow app to handle the version as appropriate.
+		msgRegisterAccount := intertxtypes.NewMsgRegisterAccount(controllerAccount.Bech32Address(chainA.Config().Bech32Prefix), ibctesting.FirstConnectionID, version)
+		err := s.RegisterInterchainAccount(ctx, chainA, controllerAccount, msgRegisterAccount)
+		s.Require().NoError(err)
+	})
+
+	t.Run("start relayer", func(t *testing.T) {
+		s.StartRelayer(relayer)
+	})
+
+	var channelOutput ibc.ChannelOutput
+	t.Run("verify interchain account", func(t *testing.T) {
+		var err error
+		interchainAcc, err = s.QueryInterchainAccount(ctx, chainA, controllerAccount.Bech32Address(chainA.Config().Bech32Prefix), ibctesting.FirstConnectionID)
+		s.Require().NoError(err)
+		s.Require().NotZero(len(interchainAcc))
+
+		channels, err := relayer.GetChannels(ctx, s.GetRelayerExecReporter(), chainA.Config().ChainID)
+		s.Require().NoError(err)
+		s.Require().Equal(len(channels), 2)
+
+		// interchain accounts channel at index: 0
+		channelOutput = channels[0]
```

Comment:
How do you know that the ICA channel is at index 0?

Reply:
Ah good question, @damiannolan do you recall why it is at index 0 rather than 1?

Reply:
Originally I would've expected it to be at index 1 (as the transfer channel is created first), but after trying that and having a failing test, I quickly switched it around and had no failures after. 
It's not ideal and I haven't dug into it much more than taking a look at this function https://github.com/strangelove-ventures/ibctest/blob/main/relayer/rly/cosmos_relayer.go#L247

Reply:
Just looking into it here, it looks like this is output of that variable

```
CHANNELS=[{State:STATE_OPEN Ordering:ORDER_ORDERED Counterparty:{PortID:icahost ChannelID:channel-1} ConnectionHops:[connection-0] Version:{"fee_version":"ics29-1","app_version":"{\"version\":\"ics27-1\",\"controller_connection_id\":\"connection-0\",\"host_connection_id\":\"connection-0\",\"address\":\"cosmos1vlmvwtdcwrdczsn97t6w8dezegvp3fpsvml28d2fx6xz8k8rpzuqqg6lr4\",\"encoding\":\"proto3\",\"tx_type\":\"sdk_multi_msg\"}"} PortID:icacontroller-cosmos1qjkpfk26cfftrj096q759ftv4ap3lfqgatngdc ChannelID:channel-1} {State:STATE_OPEN Ordering:ORDER_UNORDERED Counterparty:{PortID:transfer ChannelID:channel-0} ConnectionHops:[connection-0] Version:ics20-1 PortID:transfer ChannelID:channel-0}]
```

The order seems unexpected but the channel we care about is definitely first.